scripts/test_migration_simple.py

```
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+"""
+Simple test script to verify migration functionality without MongoDB dependency.
+"""
+
+import sys
+from pathlib import Path
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)